tests/install/test_builder.py

```diff
@@ -205,3 +242,40 @@ def _some_long_io_op(_):
     build._threaded_map(_some_long_io_op, [])
     end = time.time()
     assert end - start < sleep_duration
+
+
+def test_correct_pt_variant_os():
+    # Check that all Linux variants return Linux
+    for linux_variant in build.OperatingSystem.LINUX.value:
+        os_ = build.OperatingSystem.from_str(linux_variant)
+        assert isinstance(
+            build.choose_pt_variant(os_, "x86_64", "cpu", RAI_VERSIONS.torch),
+            build._PTArchiveLinux,
+        )
+    # Check that ARM64 and X86_64 Mac OSX return the Mac variant
+    all_archs = (build.Architecture.ARM64, build.Architecture.X64)
+    for arch in all_archs:
+        os_ = build.OperatingSystem.DARWIN
+        assert isinstance(
+            build.choose_pt_variant(os_, arch, "cpu", RAI_VERSIONS.torch),
+            build._PTArchiveMacOSX,
+        )
+
+
+def test_PTArchive_MacOSX_url():
```

Comment:
Guessing this is hold over from the class name change. Maybe
```suggestion
def test_PTArchiveMacOSX_url():
```
or 
```suggestion
def test_pt_archive_macosx_url():
```